### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by winged**
Update all that's updateable at the moment

### CODE REVIEW

**Review by fugal-dy — APPROVED** [review]
nitpicked it. not sure about the policy

all good

File: pyproject.toml [hunk 1]
```diff
@@ -47,12 +47,12 @@ graphql-core = "~3.1.7"
 black = "22.10.0"
 django-extensions = "3.2.1"
 factory-boy = "3.2.1"
-Faker = "15.3.1"
+Faker = "^15.3.1"
 flake8 = "5.0.4"
 flake8-bugbear = "22.10.27"
 flake8-debugger = "4.1.2"
 flake8-docstrings = "1.6.0"
-flake8-isort = "5.0.0"
```
[the remark this answers is not in the document]
Comment (winged) [reply pyproject.toml:55]: Yes this is intended. Might do it for other deps as well in the future. I mean as long as it's backwards compatible, I don't see why we should artificially restrict ourselves

[the remark this answers is not in the document]
Comment (fugal-dy) [reply pyproject.toml:55]: fair enough

**Review by fugal-dy — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anehx) [reply pyproject.toml:55]: Actually, we once decided to only use ranges for real dependencies and exact versions for dev dependencies in order to make sure the actually installed version is clear. We have that policy in our JS projects as well and we should keep that consistently.

**Comment by anehx** [comment]
I made a PR #1940 that updates some more dependencies and also removes the deprecated dev dependencies grouping. Also, `pytest-xdist` is currently not upgradeable.

**Comment by winged** [comment]
> I made a PR #1940 that updates some more dependencies and also removes the deprecated dev dependencies grouping. Also, `pytest-xdist` is currently not upgradeable.

Oh, I completely forgot about that policy! Would make sense to put this into a comment in `pyproject.toml`, for the next guy

**Comment by winged** [comment]
Closing in favor of #1940

**Pull request closed without merging**